R/plotScoreHeatmap.R

```diff
@@ -410,11 +414,21 @@ plotScoreHeatmap <- function(results, cells.use = NULL, labels.use = NULL,
         }
     }
 
-    # Trim by labels (max.labels) & normalize
-    # Determine labels to show based on 'max.labels' with the highest
-    # pre-normalized scores, normalize, then remove other labels.
-    maxs <- rowMaxs(scale(t(scores)))
-    to.keep <- order(maxs,decreasing=TRUE)[seq_len(max.labels)]
+    ## Trim by labels (max.labels) & normalize
+    # Determine labels to show based on 'max.labels' with the highest...
+    if (any(is.na(scores))) {
```

Comment:
Should this be `!any`?

Reply:
Yes. How did my tests pass !?

Reply:
I suppose my simple test case labels were too extreme.  Anyway, there's a different bug to fix now too.

Reply:
FYI, `which.max` gives `integer(0)` instead of `NA` when there are no non-NA values.  And there are zero additional inputs for controlling this.